### PR TITLE
fix: ux: #21957 don't state that conversion...

### PR DESCRIPTION
**Pull request by nickboldt**
### What does this PR do?

fix: ux: #21957 don't state that conversion happens if we're loading the default config (it's already version devfile.schemaVersion)

Change-Id: Ica802dc0ac81c73c7da8c0a5e61a3db22cdf1812
Signed-off-by: Nick Boldt <nboldt@redhat.com>

### Screenshot/screencast of this PR
N/A

### What issues does this PR fix or reference?
N/A (or see commit message above for issue number)

### How to test this PR?
N/A

### PR Checklist

[As the author of this Pull Request I made sure that:](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#pull-request-template-and-its-checklist)

- [x] [The Eclipse Contributor Agreement is valid](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#the-eclipse-contributor-agreement-is-valid)
- [x] [Code produced is complete](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#code-produced-is-complete)
- [x] [Code builds without errors](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#code-builds-without-errors)
- [ ] [Tests are covering the bugfix](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#tests-are-covering-the-bugfix)
- [x] [The repository devfile is up to date and works, if one exists](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#the-repository-devfile-is-up-to-date-and-works)
- [x] [Sections What issues does this PR fix or reference and How to test this PR completed](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#sections-what-issues-does-this-pr-fix-or-reference-and-how-to-test-this-pr-completed)
- [x] [Relevant user documentation updated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#relevant-contributing-documentation-updated)
- [x] [Relevant contributing documentation updated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#relevant-contributing-documentation-updated)
- [x] [CI/CD changes implemented, documented and communicated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#cicd-changes-implemented-documented-and-communicated)

### Reviewers

Reviewers, please comment how you tested the PR when approving it.

### CODE REVIEW

**Review by olexii4 — APPROVED** [review]
LGTM

**Comment by olexii4** [comment]
@nickboldt You need to fix build errors.

**Comment by nickboldt** [comment]
>  You need to fix build errors.

I love that an unused variable is just a warning, but updating the copyright to the current year fails the build. 

Do we need to update all files to 2023 so that the header check passes?

File: packages/dashboard-frontend/src/containers/Loader/Factory/Steps/Fetch/Devfile/buildStepTitle.ts [hunk 1]
```diff
@@ -29,10 +29,9 @@ export default function (
     newTitle = `Devfile could not be found in ${sourceUrl}. Applying the default configuration.`;
   } else {
     newTitle = `Devfile found in repo ${sourceUrl} as '${source}'.`;
-  }
-  if (factoryResolverConverted.isConverted) {
-    const devfile = factoryResolverConverted.devfileV2;
```
Comment (ibuziuk) [inline packages/dashboard-frontend/src/containers/Loader/Factory/Steps/Fetch/Devfile/buildStepTitle.ts:34]: why can't we use `factoryResolverConverted.devfileV2` ?

Comment (nickboldt) [reply packages/dashboard-frontend/src/containers/Loader/Factory/Steps/Fetch/Devfile/buildStepTitle.ts:34]: or could `factoryResolverConverted.devfileV2.schemaVersion` sometimes contain 2.2.0 (or else 2.1.0) ?

**Pull request merged**